.moai/specs/SPEC-WORKTREE-002/spec.md

```
@@ -0,0 +1,355 @@
+# SPEC-WORKTREE-002: Worktree End-to-End Automation
+
+---
+id: SPEC-WORKTREE-002
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)